re.rizin.cutter.plugin.rz-ghidra.yml

```diff
@@ -0,0 +1,31 @@
+id: re.rizin.cutter.plugin.rz-ghidra
+branch: stable
+runtime: re.rizin.cutter
+runtime-version: stable
+appstream-compose: false
+sdk: org.kde.Sdk//5.15-21.08
```

Comment:
the app use `5.15-22.08`

Reply:
forgot to update to it after updating cutter, thanks !